src/Shared/Recruit.Vacancies.Client/Infrastructure/Repositories/MongoDbVacancyReviewRepository.cs

```diff
@@ -102,5 +114,36 @@ public Task<List<VacancyReview>> GetForVacancyAsync(long vacancyReference)
                 .Find(filter)
                 .ToListAsync(), new Context(nameof(GetForVacancyAsync)));
         }
+
+        public async Task<int> GetApprovedFirstTimeCountAsync(string submittedByUserId)
```

Comment:
This is the subset of the Approved query, perhaps consider getting a list and doing the count from in memory list instead of two different queries.

Reply:
I don't have enough info on performance to know which will be quicker. My thinking is returning a count is quicker than a projection so that's what I've gone for.  Suggest we just monitor the page load times as the data grows.